front/src/common/helper.ts

```diff
@@ -7,6 +10,36 @@ export function toMacroCase(string: string) {
     .split("")
     .map(c => (c.toUpperCase() === c ? `_${c.trim()}` : c))
     .join("")
-    .replace(/_+/g, '_')
+    .replace(/_+/g, "_")
     .toUpperCase();
 }
+
+export function uploadApolloCache<T>(
```

Comment:
est-ce que ça ne devrait pas être `updateApolloCache` plutôt ?

Reply:
Yes, of course :)